lib/iris/tests/test_analysis.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2010 - 2014, Met Office
+# (C) British Crown Copyright 2010 - 2015, Met Office
```

Comment:
It is because you included both a commit which changed the file, and a commit which reverted the change, so in the eyes of git the file has indeed been modified in 2015. The solution is to remove that change entirely, usually via rebase.